ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -54,8 +54,7 @@ azure4) export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.jso
 gcp) export GOOGLE_CLOUD_KEYFILE_JSON=${CLUSTER_PROFILE_DIR}/gce.json;;
 kubevirt) export KUBEVIRT_KUBECONFIG=${HOME}/.kube/config;;
 vsphere) ;;
-openstack) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
-openstack-vexxhost) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
+openstack*) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
```

Comment:
Let's hope it's not going to break CI for `openstack-osuosl` and `openstack-ppc64le`:

https://github.com/shiftstack/ci-tools/blob/a5c19f47eb623e7cfaabc31551f09e0c4604a52e/pkg/api/types.go#L976-L980

Reply:
Good catch! PTAL

Reply:
Can you either replicate your change to the ci-operator/step-registry/openshift/e2e/ovirt/minimal/test/openshift-e2e-ovirt-minimal-test-commands.sh file, or unstage the change altogether?